### PR TITLE
Little bit more location atmosphere.

### PR DESCRIPTION
**Pull request by cosmiccoincidence**
Adds..
- Mailboxes (furniture) as well as adds them to various residential locations.
- Coat racks (furniture & construction recipe) as well as adds them to some homes and stores.
- Stools (furniture & construction recipe) as well as adds them to some homes and stores.
- Mail (item group)
- Coat Rack (item group)
- Updated other item groups
- Laundry Basket (item)

Also compresses many lines in files within the mapgen folder that take up unneeded space (basically linted many lines but not whole files, hence why 150+ files have changed).

### CODE REVIEW

**Comment by cosmiccoincidence** [comment]
Thanks, I already have that tool. I think my issue was the tab/4space thing which I think I changed in preferences of my editor.

File: data/json/mapgen/cosmics_additions/LANCenter.json [hunk 1]
```diff
@@ -17,9 +17,13 @@
         "s": "f_null",
         "v": "f_vending_c",
         "w": "f_null",
+        "Y": "f_rack_coat",
         "{": "f_fridge",
         "|": "f_null"
       },
+      "items": {
+        "Y": { "item": "coat_rack", "chance": 40, "repeat": 2 }
+      },
```
Comment (Night-Pryanik) [inline data/json/mapgen/cosmics_additions/LANCenter.json]: This line (and most others) is short enough to be condensed to 

```
"items": { "Y": { "item": "coat_rack", "chance": 40, "repeat": 2 } }

```

Web-linter helps with this kind of issues.

File: data/json/mapgen/cosmics_additions/Cabin.json [hunk 2]
```diff
@@ -17,12 +17,13 @@
...
         "c": "f_chair",
         "f": "f_null",
+        "Y": "f_rack_coat",
         "s": "f_sink",
         "t": "f_table",
         "w": "f_null",
         "|": "f_null"
       },
-      "items": { "#": { "item": "bed", "chance": 20 }, "t": { "item": "dining", "chance": 15 } },
+      "items": { "#": { "item": "bed", "chance": 20 }, "t": { "item": "dining", "chance": 15 }, "Y": { "item": "coat_rack", "chance": 30 } },
```
Comment (Night-Pryanik) [inline data/json/mapgen/cosmics_additions/Cabin.json]: This line is too long, so it should be unfolded like

```
"items": {
  "#": { "item": "bed", "chance": 20 },
  "t": { "item": "dining", "chance": 15 },
  "Y": { "item": "coat_rack", "chance": 30 }
}
```

**Comment by ZhilkinSerg** [comment]
You have a lot of errors - both syntax and formatting.

If you use http://dev.narc.ro/cataclysm/format.html it would print `Syntax error` in case json syntax is incorrect.

You can also find out where error is using game - it will tell you error details when trying to start or load game, e.g.:

image

**Comment by ZhilkinSerg** [comment]
Some of the errors:

```diff
diff --git a/data/json/mapgen/bar.json b/data/json/mapgen/bar.json
index 71c4dd6c4b..38b5a2da9e 100644
--- a/data/json/mapgen/bar.json
+++ b/data/json/mapgen/bar.json
@@ -148,7 +148,10 @@
         "|tllls|          |lcccc|",
         "|-----|          |-----|"
       ],
-      "set": [ { "point": "terrain", "id": "t_pavement", "x": 10, "y": 21 }, { "point": "terrain", "id": "t_pavement", "x": 11, "y": 21 } ],
+      "set": [
+        { "point": "terrain", "id": "t_pavement", "x": 10, "y": 21 },
+        { "point": "terrain", "id": "t_pavement", "x": 11, "y": 21 }
+      ],
       "terrain": {
         " ": "t_pavement",
         "&": "t_linoleum_gray",
diff --git a/data/json/mapgen/bunker.json b/data/json/mapgen/bunker.json
index 7521c6bfde..f6872e1fc0 100644
--- a/data/json/mapgen/bunker.json
+++ b/data/json/mapgen/bunker.json
@@ -314,7 +314,11 @@
   {
     "type": "item_group",
     "id": "bunker_basement_books",
-    "items": [ { "group": "book_military", "prob": 50 }, { "group": "book_survival", "prob": 50 }, { "group": "magazines", "prob": 80 } ]
+    "items": [
+      { "group": "book_military", "prob": 50 },
+      { "group": "book_survival", "prob": 50 },
+      { "group": "magazines", "prob": 80 }
+    ]
   },
   {
     "type": "item_group",
diff --git a/data/json/mapgen/campsite.json b/data/json/mapgen/campsite.json
index 7e4e5350bf..41e97b755c 100644
--- a/data/json/mapgen/campsite.json
+++ b/data/json/mapgen/campsite.json
@@ -260,7 +260,7 @@
         { "item": "map_extra_college_lake", "x": [ 15, 3 ], "y": [ 20, 8 ], "chance": 75, "repeat": [ 3, 5 ] },
         { "item": "camping", "x": [ 15, 3 ], "y": [ 20, 8 ], "chance": 75 }
       ],
-      "place_monsters":  { "monster": "CAMPERS", "x": [ 0, 23 ], "y": [ 0, 23 ], "chance": 75, "repeat": [ 1, 3 ] } ],
+      "place_monsters": [ { "monster": "CAMPERS", "x": [ 0, 23 ], "y": [ 0, 23 ], "chance": 75, "repeat": [ 1, 3 ] } ],
       "place_vehicles": [
         { "vehicle": "motorcycle", "x": 3, "y": 3, "chance": 50, "rotation": 0 },
         { "vehicle": "motorcycle", "x": 17, "y": 3, "chance": 25, "rotation": 180 },
diff --git a/data/json/mapgen/cosmics_additions/Cabin.json b/data/json/mapgen/cosmics_additions/Cabin.json
index 367944e0e4..d84d593e73 100644
--- a/data/json/mapgen/cosmics_additions/Cabin.json
+++ b/data/json/mapgen/cosmics_additions/Cabin.json
@@ -23,10 +23,10 @@
         "w": "f_null",
         "|": "f_null"
       },
-      "items": { 
+      "items": {
         "#": { "item": "bed", "chance": 20 },
-        "t": { "item": "dining", "chance": 15 }, 
-        "Y": { "item": "coat_rack", "chance": 30 } 
+        "t": { "item": "dining", "chance": 15 },
+        "Y": { "item": "coat_rack", "chance": 30 }
       },
       "place_items": [
         { "chance": 20, "item": "kitchen", "x": 11, "y": 10 },
diff --git a/data/json/mapgen/fortified_house01.json b/data/json/mapgen/fortified_house01.json
index a1b5a6c8b8..d9b9d3f690 100644
--- a/data/json/mapgen/fortified_house01.json
+++ b/data/json/mapgen/fortified_house01.json
@@ -99,7 +99,10 @@
         "l": "f_stool"
       },
       "toilets": { ";": {  } },
-      "items": {  "Y": { "item": "coat_rack", "chance": 35, "repeat": [ 1, 4 ] }, "P": { "item": "mail", "chance": 20, "repeat": [ 1, 5 ] } },
+      "items": {
+        "Y": { "item": "coat_rack", "chance": 35, "repeat": [ 1, 4 ] },
+        "P": { "item": "mail", "chance": 20, "repeat": [ 1, 5 ] }
+      },
       "place_loot": [
         { "group": "bed", "x": [ 13, 14 ], "y": [ 18, 19 ], "chance": 80 },
         { "group": "survivor_weapons", "x": [ 5, 6 ], "y": [ 12, 12 ], "chance": 60, "ammo": 50, "magazine": 100 },
diff --git a/data/json/mapgen/house/house09.json b/data/json/mapgen/house/house09.json
index 95d91ffb93..9525a66563 100644
--- a/data/json/mapgen/house/house09.json
+++ b/data/json/mapgen/house/house09.json
@@ -89,11 +89,11 @@
         "r": "f_rack"
       },
       "toilets": { "t": {  } },
-      "items": { 
+      "items": {
         "p": { "item": "mail", "chance": 20, "repeat": [ 1, 5 ] },
         "W": { "item": "laundry", "chance": 50, "repeat": [ 1, 10 ] },
         "Y": { "item": "laundry", "chance": 50, "repeat": [ 1, 10 ] }
-	  },
+      },
       "place_loot": [
         { "group": "bed", "x": [ 2, 3 ], "y": [ 18, 19 ], "chance": 90, "repeat": [ 1, 3 ] },
         { "group": "livingroom", "x": [ 12, 21 ], "y": [ 2, 6 ], "chance": 90, "repeat": [ 1, 4 ] },
diff --git a/data/json/mapgen/house/house12.json b/data/json/mapgen/house/house12.json
index 9e35316052..63a590de1f 100644
--- a/data/json/mapgen/house/house12.json
+++ b/data/json/mapgen/house/house12.json
@@ -97,11 +97,11 @@
         "o": "f_bookcase"
       },
       "toilets": { "t": {  } },
-      "items": { 
+      "items": {
         "p": { "item": "mail", "chance": 20, "repeat": [ 1, 5 ] },
         "Y": { "item": "laundry", "chance": 50, "repeat": [ 1, 10 ] },
         "W": { "item": "laundry", "chance": 50, "repeat": [ 1, 10 ] }
-	  },
+      },
       "place_loot": [
         { "group": "dresser", "x": [ 19, 19 ], "y": [ 6, 6 ], "chance": 75, "repeat": [ 1, 4 ] },
         { "group": "dresser", "x": [ 13, 13 ], "y": [ 9, 9 ], "chance": 85, "repeat": [ 1, 2 ] },
diff --git a/data/json/mapgen/house/house15.json b/data/json/mapgen/house/house15.json
index 301cd87fd3..ab7a76e744 100644
--- a/data/json/mapgen/house/house15.json
+++ b/data/json/mapgen/house/house15.json
@@ -107,7 +107,7 @@
         "n": "f_table",
         "o": "f_bookcase"
       },
-      "toilets": { "t": { } },
+      "toilets": { "t": {  } },
       "place_loot": [
         { "group": "dresser", "x": [ 2, 2 ], "y": [ 17, 17 ], "chance": 90, "repeat": [ 1, 2 ] },
         { "group": "cleaning", "x": [ 18, 19 ], "y": [ 16, 16 ], "chance": 90, "repeat": [ 1, 2 ] },
diff --git a/data/json/mapgen/house/house16.json b/data/json/mapgen/house/house16.json
index 538340c02e..039cfe640b 100644
--- a/data/json/mapgen/house/house16.json
+++ b/data/json/mapgen/house/house16.json
@@ -88,12 +88,12 @@
         "n": "f_table",
         "o": "f_bookcase"
       },
-      "toilets": { "t": { } },
-      "items": { 
+      "toilets": { "t": {  } },
+      "items": {
         "p": { "item": "mail", "chance": 20, "repeat": [ 1, 5 ] },
         "W": { "item": "laundry", "chance": 50, "repeat": [ 1, 10 ] },
         "Y": { "item": "laundry", "chance": 50, "repeat": [ 1, 10 ] }
-	  },
+      },
       "place_loot": [
         { "group": "dining", "x": [ 18, 19 ], "y": [ 15, 16 ], "chance": 75, "repeat": [ 1, 4 ] },
         { "group": "oven", "x": [ 14, 14 ], "y": [ 13, 13 ], "chance": 75, "repeat": [ 1, 3 ] },
diff --git a/data/json/mapgen/house/house17.json b/data/json/mapgen/house/house17.json
index 672cdb606d..83182fd8f8 100644
--- a/data/json/mapgen/house/house17.json
+++ b/data/json/mapgen/house/house17.json
@@ -94,8 +94,11 @@
         "n": "f_table",
         "o": "f_bookcase"
       },
-      "toilets": { "t": { } },
-      "items": { "Y": { "item": "laundry", "chance": 50, "repeat": [ 1, 10 ] }, "W": { "item": "laundry", "chance": 50, "repeat": [ 1, 10 ] } },
+      "toilets": { "t": {  } },
+      "items": {
+        "Y": { "item": "laundry", "chance": 50, "repeat": [ 1, 10 ] },
+        "W": { "item": "laundry", "chance": 50, "repeat": [ 1, 10 ] }
+      },
       "place_loot": [
         { "group": "livingroom", "x": [ 10, 17 ], "y": [ 15, 19 ], "chance": 90, "repeat": [ 1, 6 ] },
         { "group": "kitchen", "x": [ 4, 7 ], "y": [ 16, 19 ], "chance": 85, "repeat": [ 1, 3 ] },
diff --git a/data/json/mapgen/house/house18.json b/data/json/mapgen/house/house18.json
index a2406d980c..77274d52d4 100644
--- a/data/json/mapgen/house/house18.json
+++ b/data/json/mapgen/house/house18.json
@@ -87,56 +87,59 @@
         "r": "f_rack",
         "u": "f_sink"
       },
-      "toilets": { "t": { } },
-      "items": { "Y": { "item": "coat_rack", "chance": 30, "repeat": [ 1, 4 ] }, "p": { "item": "mail", "chance": 20, "repeat": [ 1, 5 ] },
-      "place_loot": [
-        { "group": "homebooks", "x": [ 3, 3 ], "y": [ 16, 16 ], "chance": 70, "repeat": [ 1, 2 ] },
-        { "group": "homebooks", "x": [ 5, 5 ], "y": [ 16, 16 ], "chance": 70, "repeat": [ 1, 2 ] },
-        { "group": "homebooks", "x": [ 16, 17 ], "y": [ 16, 16 ], "chance": 70, "repeat": [ 1, 3 ] },
-        { "group": "magazines", "x": [ 4, 5 ], "y": [ 4, 5 ], "chance": 80, "repeat": [ 1, 3 ] },
-        { "group": "magazines", "x": [ 16, 17 ], "y": [ 2, 2 ], "chance": 90, "repeat": [ 1, 4 ] },
-        { "group": "novels", "x": [ 10, 10 ], "y": [ 5, 5 ], "chance": 85, "repeat": [ 1, 3 ] },
-        { "group": "office", "x": [ 17, 19 ], "y": [ 22, 22 ], "chance": 80, "repeat": [ 1, 3 ] },
-        { "group": "livingroom", "x": [ 2, 6 ], "y": [ 3, 8 ], "chance": 90, "repeat": [ 1, 5 ] },
-        { "group": "dining", "x": [ 17, 19 ], "y": [ 5, 6 ], "chance": 85, "repeat": [ 1, 3 ] },
-        { "group": "dining", "x": [ 4, 4 ], "y": [ 14, 14 ], "chance": 70, "repeat": [ 1, 5 ] },
-        { "group": "dining", "x": [ 19, 19 ], "y": [ 14, 14 ], "chance": 70, "repeat": [ 1, 4 ] },
-        { "group": "fridge", "x": [ 5, 5 ], "y": [ 14, 14 ], "chance": 90, "repeat": [ 1, 4 ] },
-        { "group": "fridge", "x": [ 18, 18 ], "y": [ 10, 10 ], "chance": 90, "repeat": [ 1, 3 ] },
-        { "group": "oven", "x": [ 2, 2 ], "y": [ 12, 12 ], "chance": 75, "repeat": [ 1, 3 ] },
-        { "group": "oven", "x": [ 20, 20 ], "y": [ 12, 12 ], "chance": 75, "repeat": [ 1, 3 ] },
-        { "group": "kitchen", "x": [ 2, 3 ], "y": [ 10, 14 ], "chance": 85, "repeat": [ 1, 3 ] },
-        { "group": "kitchen", "x": [ 19, 20 ], "y": [ 10, 14 ], "chance": 85, "repeat": [ 1, 3 ] },
-        { "group": "dresser", "x": [ 2, 2 ], "y": [ 18, 19 ], "chance": 85, "repeat": [ 1, 5 ] },
-        { "group": "dresser", "x": [ 2, 2 ], "y": [ 22, 22 ], "chance": 85, "repeat": [ 1, 3 ] },
-        { "group": "allclothes", "x": [ 10, 10 ], "y": [ 11, 11 ], "chance": 30, "repeat": [ 1, 4 ] },
-        { "group": "allclothes", "x": [ 12, 12 ], "y": [ 11, 11 ], "chance": 30, "repeat": [ 1, 4 ] },
-        { "group": "allclothes", "x": [ 10, 10 ], "y": [ 22, 22 ], "chance": 95, "repeat": [ 1, 2 ] },
-        { "group": "allclothes", "x": [ 12, 12 ], "y": [ 22, 22 ], "chance": 95, "repeat": [ 1, 3 ] },
-        { "group": "dresser", "x": [ 20, 20 ], "y": [ 17, 17 ], "chance": 90, "repeat": [ 1, 3 ] },
-        { "group": "dresser", "x": [ 20, 20 ], "y": [ 20, 20 ], "chance": 90, "repeat": [ 1, 3 ] },
-        { "group": "softdrugs", "x": [ 8, 10 ], "y": [ 19, 20 ], "chance": 75, "repeat": [ 1, 3 ] },
-        { "group": "softdrugs", "x": [ 12, 14 ], "y": [ 19, 20 ], "chance": 75, "repeat": [ 1, 3 ] },
-        { "group": "harddrugs", "x": [ 8, 10 ], "y": [ 19, 20 ], "chance": 15, "repeat": [ 1, 2 ] },
-        { "group": "harddrugs", "x": [ 12, 14 ], "y": [ 19, 20 ], "chance": 15, "repeat": [ 1, 2 ] },
-        { "group": "cleaning", "x": [ 8, 10 ], "y": [ 19, 20 ], "chance": 90, "repeat": [ 1, 2 ] },
-        { "group": "cleaning", "x": [ 12, 14 ], "y": [ 19, 20 ], "chance": 90, "repeat": [ 1, 2 ] },
-        { "group": "guns_common", "x": [ 10, 10 ], "y": [ 13, 13 ], "chance": 10, "ammo": 85, "magazine": 95 },
-        { "group": "guns_common", "x": [ 12, 12 ], "y": [ 13, 13 ], "chance": 10, "ammo": 85, "magazine": 95 },
-        { "group": "home_hw", "x": [ 10, 10 ], "y": [ 13, 13 ], "chance": 80, "repeat": [ 1, 3 ] },
-        { "group": "home_hw", "x": [ 12, 12 ], "y": [ 13, 13 ], "chance": 80, "repeat": [ 1, 3 ] },
-        { "group": "cannedfood", "x": [ 10, 10 ], "y": [ 13, 13 ], "chance": 70, "repeat": [ 1, 4 ] },
-        { "group": "cannedfood", "x": [ 12, 12 ], "y": [ 13, 13 ], "chance": 70, "repeat": [ 1, 3 ] },
-        { "group": "jackets", "x": [ 10, 10 ], "y": [ 4, 4 ], "chance": 70, "repeat": [ 1, 2 ] },
-        { "group": "jackets", "x": [ 12, 12 ], "y": [ 5, 5 ], "chance": 70, "repeat": [ 1, 2 ] },
-        { "group": "bed", "x": [ 3, 4 ], "y": [ 21, 22 ], "chance": 90, "repeat": [ 1, 3 ] },
-        { "group": "bed", "x": [ 19, 20 ], "y": [ 18, 19 ], "chance": 90, "repeat": [ 1, 3 ] },
-        { "group": "bedroom", "x": [ 2, 6 ], "y": [ 16, 22 ], "chance": 80, "repeat": [ 1, 5 ] },
-        { "group": "bedroom", "x": [ 16, 20 ], "y": [ 16, 22 ], "chance": 80, "repeat": [ 1, 5 ] },
-        { "item": "television", "x": [ 4, 4 ], "y": [ 2, 2 ], "chance": 70 },
-        { "item": "television", "x": [ 4, 4 ], "y": [ 16, 16 ], "chance": 10 }
-      ],
-      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "chance": 8 } ]
+      "toilets": { "t": {  } },
+      "items": {
+        "Y": { "item": "coat_rack", "chance": 30, "repeat": [ 1, 4 ] },
+        "p": { "item": "mail", "chance": 20, "repeat": [ 1, 5 ] },
+        "place_loot": [
+          { "group": "homebooks", "x": [ 3, 3 ], "y": [ 16, 16 ], "chance": 70, "repeat": [ 1, 2 ] },
+          { "group": "homebooks", "x": [ 5, 5 ], "y": [ 16, 16 ], "chance": 70, "repeat": [ 1, 2 ] },
+          { "group": "homebooks", "x": [ 16, 17 ], "y": [ 16, 16 ], "chance": 70, "repeat": [ 1, 3 ] },
+          { "group": "magazines", "x": [ 4, 5 ], "y": [ 4, 5 ], "chance": 80, "repeat": [ 1, 3 ] },
+          { "group": "magazines", "x": [ 16, 17 ], "y": [ 2, 2 ], "chance": 90, "repeat": [ 1, 4 ] },
+          { "group": "novels", "x": [ 10, 10 ], "y": [ 5, 5 ], "chance": 85, "repeat": [ 1, 3 ] },
+          { "group": "office", "x": [ 17, 19 ], "y": [ 22, 22 ], "chance": 80, "repeat": [ 1, 3 ] },
+          { "group": "livingroom", "x": [ 2, 6 ], "y": [ 3, 8 ], "chance": 90, "repeat": [ 1, 5 ] },
+          { "group": "dining", "x": [ 17, 19 ], "y": [ 5, 6 ], "chance": 85, "repeat": [ 1, 3 ] },
+          { "group": "dining", "x": [ 4, 4 ], "y": [ 14, 14 ], "chance": 70, "repeat": [ 1, 5 ] },
+          { "group": "dining", "x": [ 19, 19 ], "y": [ 14, 14 ], "chance": 70, "repeat": [ 1, 4 ] },
+          { "group": "fridge", "x": [ 5, 5 ], "y": [ 14, 14 ], "chance": 90, "repeat": [ 1, 4 ] },
+          { "group": "fridge", "x": [ 18, 18 ], "y": [ 10, 10 ], "chance": 90, "repeat": [ 1, 3 ] },
+          { "group": "oven", "x": [ 2, 2 ], "y": [ 12, 12 ], "chance": 75, "repeat": [ 1, 3 ] },
+          { "group": "oven", "x": [ 20, 20 ], "y": [ 12, 12 ], "chance": 75, "repeat": [ 1, 3 ] },
+          { "group": "kitchen", "x": [ 2, 3 ], "y": [ 10, 14 ], "chance": 85, "repeat": [ 1, 3 ] },
+          { "group": "kitchen", "x": [ 19, 20 ], "y": [ 10, 14 ], "chance": 85, "repeat": [ 1, 3 ] },
+          { "group": "dresser", "x": [ 2, 2 ], "y": [ 18, 19 ], "chance": 85, "repeat": [ 1, 5 ] },
+          { "group": "dresser", "x": [ 2, 2 ], "y": [ 22, 22 ], "chance": 85, "repeat": [ 1, 3 ] },
+          { "group": "allclothes", "x": [ 10, 10 ], "y": [ 11, 11 ], "chance": 30, "repeat": [ 1, 4 ] },
+          { "group": "allclothes", "x": [ 12, 12 ], "y": [ 11, 11 ], "chance": 30, "repeat": [ 1, 4 ] },
+          { "group": "allclothes", "x": [ 10, 10 ], "y": [ 22, 22 ], "chance": 95, "repeat": [ 1, 2 ] },
+          { "group": "allclothes", "x": [ 12, 12 ], "y": [ 22, 22 ], "chance": 95, "repeat": [ 1, 3 ] },
+          { "group": "dresser", "x": [ 20, 20 ], "y": [ 17, 17 ], "chance": 90, "repeat": [ 1, 3 ] },
+          { "group": "dresser", "x": [ 20, 20 ], "y": [ 20, 20 ], "chance": 90, "repeat": [ 1, 3 ] },
+          { "group": "softdrugs", "x": [ 8, 10 ], "y": [ 19, 20 ], "chance": 75, "repeat": [ 1, 3 ] },
+          { "group": "softdrugs", "x": [ 12, 14 ], "y": [ 19, 20 ], "chance": 75, "repeat": [ 1, 3 ] },
+          { "group": "harddrugs", "x": [ 8, 10 ], "y": [ 19, 20 ], "chance": 15, "repeat": [ 1, 2 ] },
+          { "group": "harddrugs", "x": [ 12, 14 ], "y": [ 19, 20 ], "chance": 15, "repeat": [ 1, 2 ] },
+          { "group": "cleaning", "x": [ 8, 10 ], "y": [ 19, 20 ], "chance": 90, "repeat": [ 1, 2 ] },
+          { "group": "cleaning", "x": [ 12, 14 ], "y": [ 19, 20 ], "chance": 90, "repeat": [ 1, 2 ] },
+          { "group": "guns_common", "x": [ 10, 10 ], "y": [ 13, 13 ], "chance": 10, "ammo": 85, "magazine": 95 },
+          { "group": "guns_common", "x": [ 12, 12 ], "y": [ 13, 13 ], "chance": 10, "ammo": 85, "magazine": 95 },
+          { "group": "home_hw", "x": [ 10, 10 ], "y": [ 13, 13 ], "chance": 80, "repeat": [ 1, 3 ] },
+          { "group": "home_hw", "x": [ 12, 12 ], "y": [ 13, 13 ], "chance": 80, "repeat": [ 1, 3 ] },
+          { "group": "cannedfood", "x": [ 10, 10 ], "y": [ 13, 13 ], "chance": 70, "repeat": [ 1, 4 ] },
+          { "group": "cannedfood", "x": [ 12, 12 ], "y": [ 13, 13 ], "chance": 70, "repeat": [ 1, 3 ] },
+          { "group": "jackets", "x": [ 10, 10 ], "y": [ 4, 4 ], "chance": 70, "repeat": [ 1, 2 ] },
+          { "group": "jackets", "x": [ 12, 12 ], "y": [ 5, 5 ], "chance": 70, "repeat": [ 1, 2 ] },
+          { "group": "bed", "x": [ 3, 4 ], "y": [ 21, 22 ], "chance": 90, "repeat": [ 1, 3 ] },
+          { "group": "bed", "x": [ 19, 20 ], "y": [ 18, 19 ], "chance": 90, "repeat": [ 1, 3 ] },
+          { "group": "bedroom", "x": [ 2, 6 ], "y": [ 16, 22 ], "chance": 80, "repeat": [ 1, 5 ] },
+          { "group": "bedroom", "x": [ 16, 20 ], "y": [ 16, 22 ], "chance": 80, "repeat": [ 1, 5 ] },
+          { "item": "television", "x": [ 4, 4 ], "y": [ 2, 2 ], "chance": 70 },
+          { "item": "television", "x": [ 4, 4 ], "y": [ 16, 16 ], "chance": 10 }
+        ],
+        "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 2, 21 ], "y": [ 2, 21 ], "chance": 8 } ]
+      }
     }
   }
 ]
diff --git a/data/json/mapgen/house/house_garage5.json b/data/json/mapgen/house/house_garage5.json
index 4f0120e1b6..a78b2f5b4e 100644
--- a/data/json/mapgen/house/house_garage5.json
+++ b/data/json/mapgen/house/house_garage5.json
@@ -105,4 +105,5 @@
       "place_vehicles": [ { "vehicle": "suburban_home", "x": 5, "y": 5, "chance": 10, "rotation": 270 } ]
     }
   }
+  }
 ]
diff --git a/data/json/mapgen/house/house_suicide.json b/data/json/mapgen/house/house_suicide.json
index 7c8f4c4b7e..d524d35c1f 100644
--- a/data/json/mapgen/house/house_suicide.json
+++ b/data/json/mapgen/house/house_suicide.json
@@ -86,6 +86,7 @@
       ],
       "place_nested": [ { "chunks": [ [ "bathroom_suicide", 100 ] ], "x": 18, "y": 13 } ]
     }
+  }
   },
   {
     "id": "house_suicide_clothing",
diff --git a/data/json/mapgen/motel.json b/data/json/mapgen/motel.json
index 560abf7e2d..76b481a875 100644
--- a/data/json/mapgen/motel.json
+++ b/data/json/mapgen/motel.json
@@ -107,7 +107,7 @@
         "F": "f_fridge",
         "G": "f_oven"
       },
-      "items": { "Y": { "item": "coat_rack", "chance": 35, "repeat": [ 1, 4 ] },
+      "items": { "Y": { "item": "coat_rack", "chance": 35, "repeat": [ 1, 4 ] } },
       "place_items": [ { "item": "traveler", "x": [ 6, 6 ], "y": [ 6, 6 ], "chance": 90 } ],
       "add": [
         { "item": "roadmap", "x": 8, "y": 8, "chance": 4 },
```

**Comment by Mecares** [comment]
https://www.reddit.com/r/cataclysmdda/comments/95vg88/psa_deconstructing_a_coat_rack_gives_you_600_nails/

Looks like stacks of 100 nails are gained when deconstructed instead of individual nails

**Comment by ZhilkinSerg** [comment]
Thanks. Would be fixed in #24737.

**Pull request merged**